applications/nrf5340_audio/README.rst

```diff
@@ -861,6 +861,13 @@ Entering the DFU mode
 The |NCS| uses :ref:`SMP server and mcumgr <zephyr:device_mgmt>` as the DFU backend.
 Unlike the CIS and BIS modes for gateway and headsets, the DFU mode is advertising using the SMP server service.
 For this reason, to enter the DFU mode, you must long press **BTN 4** during each device startup to have the nRF5340 Audio DK enter the DFU mode.
+To distinguish the device role, DFU mode advertising name will be based on role and the pattern : ``CONFIG_BT_DEVICE_NAME`` _ **ROLE** _DFU.
```

Comment:
Quite a few questions:
1. How does this information relate to entering the DFU?
2. Where is the name displayed?
3. What is the relation of the pattern against lines 867-870.
4. When is this Kconfig set? It doesn't appear anywhere in the doc except for here.

Reply:
We decide to unify BT_DEVICE_NAME for CIS and BIS.

Reply:
Maybe we can change "CONFIG_BT_DEVICE_NAME" to just "NRF5340_AUDIO"?
(Meaning that we do not explicitly say in this doc that the name comes from this config)

Reply:
CONFIG_BT_DEVICE_NAME is configurable. Customer might change the value from "NRF5340_AUDIO" to another value.
How about add a line * Default ``CONFIG_BT_DEVICE_NAME``: NRF5340_AUDIO. Then the name pattern.

Reply:
```suggestion
To distinguish the device types, DFU mode advertising names will be based on these. The advertising names follow this pattern: NRF5340_AUDIO_ **ROLE** _DFU. The first part of the names is based on ``CONFIG_BT_DEVICE_NAME`` . 
```